domFoundations/events,storage,attributes/handlers.js

```diff
@@ -0,0 +1,8 @@
+const swBtn = document.getElementById('swichButtn');
+
+export const handlers = (...events) => {
```

Comment:
Could you name this function right? What does it do? Use a verb, and be specific.